config/router_config_test.go

```diff
@@ -28,9 +28,11 @@ import (
 
 import (
 	_ "github.com/apache/dubbo-go/cluster/router/condition"
+	"github.com/dubbogo/gost/container/set"
```

Comment:
it should be placed in the 2rd import block.

Reply:
resolved